node/packages/parser/src/ethereum/index.ts

```diff
@@ -171,6 +175,19 @@ export class TransactionParser {
       }
     })
 
+    const metaData: TxMetadata = (() => {
+      const buyTx = getBuyTx(pTx)
+      const sellTx = getSellTx(pTx)
+      return {
+        buyTx,
+        sellTx,
+      }
+    })()
+
+    if (metaData) {
```

Comment:
This will always evaluate to true as it stands

Reply:
Indeed, I got ahead of myself: https://github.com/shapeshift/unchained/pull/247/commits/f033e0557e89e877934340067a572504c431f00f